### ISSUE TITLE
Node version warning

### ISSUE THREAD

**Issue by ulfryk** [kucherenko/jscpd#36]
Installing `jscpd` plugin on newest node creates a warning:

```
npm WARN engine jscpd@0.3.6: wanted: {"node":"~0.10.x"} (current: {"node":"0.12.7","npm":"2.14.2"})
```

Please update package.json file (and plugin if needed) so it's compatible.

**Comment by kucherenko** [comment]
okey, thank you

**Comment by ulfryk** [comment]
np ;)

**Comment by kucherenko** [comment]
already fixed in 0.4.0

**Comment by ulfryk** [comment]
Sorry, for that - didn't notice that `gulp-jscpd` uses outdated version of `jscpd`.

Thanks for great work anyway :)
